site/src/sections/Chapters/Callout.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import personIcon from "../../assets/images/chapters/person.svg";
+import layer5Logo from "../../assets/images/chapters/layer5-discuss-white.png";
+
+const Callout = () => {
+  return (
+    <div className="callout">
+      <a href="https://discuss.layer5.io/">
```

Comment:
Add a `target="_blank"` and `rel=noreferrer`

Reply:
Resolved